README.md

```diff
@@ -50,12 +37,12 @@ You will see an output similar to this:
 172.28.103.186
 ```
 
-You can connect the running container using the IP address and configured port, `http://172.28.103.186:8000` in the example shown.
+You can connect the running container using the IP address and configured port, `http://172.28.103.186` in the example shown.
 
 For a comprehensive tutorial on running an ASP.NET app in a container, check out [the tutorial on the docs site](https://docs.microsoft.com/en-us/dotnet/articles/framework/docker/aspnetmvc).
 
 ## Supported Docker Versions
-This image has been tested on Docker Versions 1.12.1-beta26 or higher.
+This image has been tested on Docker Versions 1.12.2-beta or higher.
```

Comment:
This image has been tested on Docker Versions [1.12.2-beta](https://download.docker.com/components/engine/windows-server/cs-1.12/docker-1.12.2.zip) or higher